spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -35,7 +35,7 @@
 
 	<properties>
 		<gcp-libraries-bom.version>26.57.0</gcp-libraries-bom.version>
-		<cloud-sql-socket-factory.version>1.23.1</cloud-sql-socket-factory.version>
+		<cloud-sql-socket-factory.version>1.25.0</cloud-sql-socket-factory.version>
```

Comment:
@diegomarquezp I'm guessing we need this update because this is the version that includes the GraalVM 17 fix?

Reply:
Exactly. I'm adding this to the description. Thanks for pointing this otherwise mysterious change.